### PR TITLE
add faq to snowflake app per pm

### PR DESCRIPTION
**Pull request by mirnawong1**
add env var requirement to the faq per [internal slack thread](https://dbt-labs.slack.com/archives/C071SHKRA7J/p1722465672634289?thread_ts=1722335987.820679&cid=C071SHKRA7J) and PM notification.

### CODE REVIEW

File: website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md [hunk 1]
```diff
@@ -146,3 +146,8 @@ Check that the SL user has been granted access to the `dbt_sl_llm` schema and ma
 
 If there's been an update to the dbt Cloud account ID, access URL, or API service token, you need to update the configuration for the dbt Snowflake Native App. In Snowflake, navigate to the app's configuration page and delete the existing configurations. Add the new configuration and then run `CALL app_public.restart_ap ();` in the application database in Snowsight. 
 </Expandable>
+
+<Expandable alt_header="Are environment variables supported in the Native App?" >
+
+[Environment variables](/docs/build/environment-variables), like `{{env_var('DBT_WAREHOUSE') }}` aren’t supported the dbt Semantic Layer yet. You must use the actual credentials instead.
```
Comment (mirnawong1) [inline website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md]:
```suggestion
[Environment variables](/docs/build/environment-variables), like `{{env_var('DBT_WAREHOUSE') }}` aren’t supported the dbt Semantic Layer yet. To use the 'Ask dbt' feature, you must use the actual credentials instead.
```

File: website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md [hunk 2]
```diff
@@ -146,3 +146,8 @@ Check that the SL user has been granted access to the `dbt_sl_llm` schema and ma
 
 If there's been an update to the dbt Cloud account ID, access URL, or API service token, you need to update the configuration for the dbt Snowflake Native App. In Snowflake, navigate to the app's configuration page and delete the existing configurations. Add the new configuration and then run `CALL app_public.restart_ap ();` in the application database in Snowsight. 
 </Expandable>
+
+<Expandable alt_header="Are environment variables supported in the Native App?" >
+
+[Environment variables](/docs/build/environment-variables), like `{{env_var('DBT_WAREHOUSE') }}` aren’t supported the dbt Semantic Layer yet. To use the 'Ask dbt' feature, you must use the actual credentials instead.
```
Comment (nataliefiann) [inline website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md]:
```suggestion
[Environment variables](/docs/build/environment-variables), like `{{env_var('DBT_WAREHOUSE') }}` aren’t supported in the dbt Semantic Layer yet. To use the 'Ask dbt' feature, you must use the actual credentials instead.
```

File: website/snippets/_new-sl-setup.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ We recommend configuring credentials and service tokens to reflect your teams an
 Note that: 
 - Admins can link multiple service tokens to a single credential within a project, but each service token can only be linked to one credential per project.
 - When you send a request through the APIs, the service token of the linked credential will follow access policies of the underlying view and tables used to build your semantic layer requests.
-- [Environment variables](/docs/build/environment-variables), like `{{env_var('DBT_WAREHOUSE')}` aren't supported the dbt Semantic Layer yet. You must use the actual credentials instead.
+- [Environment variables](/docs/build/environment-variables), like `{{env_var('DBT_WAREHOUSE') }}` aren't supported the dbt Semantic Layer yet. You must use the actual credentials instead.
```
Comment (nataliefiann) [inline website/snippets/_new-sl-setup.md]:
```suggestion
- [Environment variables](/docs/build/environment-variables), like `{{env_var('DBT_WAREHOUSE') }}` aren't supported in the dbt Semantic Layer yet. You must use the actual credentials instead.
```

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thank you for creating this PR. I made one small grammar change and committed it but other than that, no suggestions or comments from me. I've approved this for you.

Kind Regards
Natalie

**Pull request merged**